src/labelle/lib/devices/dymo_labeler.py

```diff
@@ -254,14 +254,15 @@ def __init__(
 
         if self._device is not None:
             # Retrieve device config
-            self._deviceConfig = get_device_config_by_id(self._device.id_product)
+            foundConfig = get_device_config_by_id(self._device.id_product)
+            if foundConfig is not None:
+                self._deviceConfig = foundConfig
+            else:
+                raise ValueError(f"Unsupported device type {self._device.id_product}")
```

Comment:
Why do you return `None` from the function and raise here, instead of raising in the function itself, and then you will not have to test returned value from the function (and change its type to also support `None`).

Reply:
Good question...
Stupid answer: I learned about the raising exceptions later on. 

will fix.

Reply:
Fixed